examples/iam-policy-management.v1.test.js

```diff
@@ -50,7 +50,7 @@ const originalWarn = console.warn;
 const consoleLogMock = jest.spyOn(console, 'log');
 const consoleWarnMock = jest.spyOn(console, 'warn');
 
-describe('IamPolicyManagementV1', () => {
+describe.only('IamPolicyManagementV1', () => {
```

Comment:
I'm not familiar with `.only` in this context.  Just curious what the significance is here.

Reply:
.only is used by different frameworks to run specific tests (mocha-chai), I left it in before realizing jest does not use .only after the describe and missed removing it.